config/substrate.go

```diff
@@ -0,0 +1,70 @@
+// The Licensed Work is (c) 2022 Sygma
+// SPDX-License-Identifier: BUSL-1.1
+
+package config
+
+import (
+	"math/big"
+	"time"
+
+	"github.com/creasty/defaults"
+	"github.com/mitchellh/mapstructure"
+
+	"github.com/ChainSafe/chainbridge-core/config/chain"
+)
+
+type SubstrateConfig struct {
```

Comment:
I would rather move this to substrate package. It would make more sense in the long run.